src/Summoner/Project.hs

```diff
@@ -153,10 +154,11 @@ generateProject projectName Targets{..} = do
     test   <- decisionToBool isTest "tests"
     bench  <- decisionToBool isBenchmark "benchmarks"
 
-    putStrLn "Latest GHCs: 7.10.3 8.0.2 8.2.2"
-    putStrLn $ "The project will be created with the latest resolver for GHC-" ++ T.unpack defaultGHC
-    testedVersions <- T.words <$>
+    T.putStrLn "Latest GHCs: 7.10.3 8.0.1 8.0.2 8.2.2"
+    T.putStrLn $ "The project will be created with the latest resolver for GHC-" <> showGhcVer defaultGHC
```

Comment:
Here also, as you mentioned above, it's not the latest, but default (or latest for summoner, but can be changed by user).